CrewChiefV4/ACS/ACSGameStateMapper.cs

```diff
@@ -232,17 +234,34 @@ public GameStateData mapToGameStateData(Object memoryMappedFileStruct, GameState
             bool isInPits = shared.acsGraphic.isInPit == 1;
             int lapsCompleated = shared.acsGraphic.completedLaps;
             int numberOfSectorsOnTrack = shared.acsStatic.sectorCount;
- 
+            
             int numberOfLapsInSession = (int)shared.acsGraphic.numberOfLaps;
+            
+            float gameSessionTimeLeft = 0.0f;
+            if (!Double.IsInfinity(shared.acsGraphic.sessionTimeLeft))
```

Comment:
OK. I'll merge it as-is but I think it's definitely worth spending a bit of time tracking down why it can be Infinity. It surprises me because this number is being read from a Struct - I didn't expect the struct data structure to support Infinity (after all, the number is just 4 (for a float) or 8 (for a double) bytes)